routers/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+
+const musicController = require("../controller/musicController");
+
+router.get("/home", musicController.home);
```

Comment:
Creo quedaría mejor que la raiz te muestre el home y que los usuarios no tengan que poner home para la vista de inicio 😉 
```suggestion
router.get("/", musicController.home);
```